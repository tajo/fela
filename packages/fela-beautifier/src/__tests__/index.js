import { createRenderer } from 'fela'
import { renderToString } from 'fela-tools'
import beautifier from '../index'

describe('Beautifier enhancer', () => {
  it('beautify', () => {
    const renderer = createRenderer({
      enhancers: [beautifier()]
    })
    renderer.renderRule(() => ({ color: 'blue' }))
    renderer.renderRule(() => ({ color: 'red', fontSize: '12px' }))
    expect(renderToString(renderer)).toMatchSnapshot()
  })
})
