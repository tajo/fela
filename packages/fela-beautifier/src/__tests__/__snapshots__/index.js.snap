// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Beautifier enhancer beautify 1`] = `".a{color:red}.b{font-size:12px}"`;
